app/views/layouts/_navbar.html.erb

```diff
@@ -1,8 +1,7 @@
-<nav class="navbar navbar-expand-md bg-secondary navbar-dark" id="mainNav">
+<nav class="navbar navbar-expand-md navbar-light" id="mainNav">
   <div class="container">
     <%= link_to root_path, class: "navbar-brand" do %>
       <%= image_tag image_path('favicon.svg'), size: "32", class: "d-inline-block mr-2" %>
```

Comment:
The design of this logo is... intricate, we could try to make it bigger. Just a suggestion tho